examples/todo/src/controllers/todo.controller.ts

```diff
@@ -35,7 +44,18 @@ export class TodoController {
     @param.path.number('id') id: number,
     @param.query.boolean('items') items?: boolean,
   ): Promise<Todo> {
-    return await this.todoRepo.findById(id);
+    try {
+      return await this.todoRepo.findById(id);
+    } catch (e) {
+      /*
+This is just a quick hack to demonstrate the problem.
+*/
+      if (e.message.match(/ound with id/)) {
```

Comment:
found

Reply:
> This is just a quick hack to demonstrate the problem.

😃

Reply:
We should for a solution which requires no change in controllers.